src/i18n/en.json

```diff
@@ -0,0 +1,11 @@
+{
+	"login": {
+		"title": "Welcome to\n",
+		"growName": "Grow Hub!",
+		"subTitle": "Your pocket guide for plant care",
+		"linkText": "Log in as a guest"
```

Comment:
it is a better practice to try to keep vocabulary keys closer to meaning of their direct translations. E.g here `"logAsGuest": "Log in as a guest"` would be a better option